luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/SingleDataSourcePhysicalTableFactory.java

```diff
@@ -86,7 +86,7 @@ LuthierPhysicalTableParams buildParams(
         metricsNode.forEach(
                 node -> {
                     String metricName = node.textValue();
-                    resourceFactories.getMetric(metricName);
+                    // resourceFactories.getMetric(metricName);
```

Comment:
I believe we need this statement or the construction on the Table will fail.

Reply:
This is commented out because I don't have the corresponding Factories for all the metrics referenced in the physicalTable _yet_.